framework/auth/views.py

```diff
@@ -36,10 +35,12 @@
 
 @collect_auth
 def reset_password(auth, **kwargs):
+    """ If get request, show reset password page. If POST, attempt to reset password if
```

Comment:
Part of this docstring should go to the function below.

On a related note, spoke in person about why this function was split up. :+1: from me